tests/sqlparser_common.rs

```diff
@@ -1093,6 +1093,26 @@ fn parse_unary_math() {
     );
 }
 
+#[test]
+fn parse_unary_math_with_multiply() {
+    use self::Expr::*;
+    let sql = "-a * -b";
```

Comment:
I ran this case without any of the code changes in this PR and it fails like the following (aka it is parased as `-(a * -b)` 

```
---- parse_unary_math_with_multiply stdout ----
thread 'parse_unary_math_with_multiply' panicked at 'assertion failed: `(left == right)`
  left: `BinaryOp { left: UnaryOp { op: Minus, expr: Identifier(Ident { value: "a", quote_style: None }) }, op: Multiply, right: UnaryOp { op: Minus, expr: Identifier(Ident { value: "b", quote_style: None }) } }`,
 right: `UnaryOp { op: Minus, expr: BinaryOp { left: Identifier(Ident { value: "a", quote_style: None }), op: Multiply, right: UnaryOp { op: Minus, expr: Identifier(Ident { value: "b", quote_style: None }) } } }`', tests/sqlparser_common.rs:1100:5
stack backtrace:
```